experiments/clean_csv.py

```
@@ -0,0 +1,270 @@
+    return program_id
+
+def create_payment(row, program_id, recipient_id):
+    """Create payment record."""
+    if not program_id or not recipient_id:
+        raise ArgumentError("program_id and recipient_id must be passed in")
+    
+    amount = row.get('AGRG_PYMT_AMT', '')
+    
+    # Skip if empty or not a number
+    if not amount:
+        raise "No payment amount found"
```

Ruff (0.8.2)

Cannot raise a literal. Did you intend to return it or raise an Exception?

(B016)